anda/lib/mesa/mesa.spec

```
@@ -72,14 +71,21 @@
 
 %global vulkan_drivers swrast%{?base_vulkan}%{?intel_platform_vulkan}%{?asahi_platform_vulkan}%{?extra_platform_vulkan}%{?with_nvk:,nouveau}%{?with_virtio:,virtio}%{?with_d3d12:,microsoft-experimental}
 
+%if 0%{?with_nvk} && 0%{?rhel}
+%global vendor_nvk_crates 1
```

We already vendor Crates.
```suggestion
%global vendor_nvk_crates 0
```

---

From Fedora, no change